StripeConnect/StripeConnect/Source/Internal/Webview/ConnectComponentWebViewController.swift

```diff
@@ -49,13 +53,15 @@ class ConnectComponentWebViewController: ConnectWebViewController {
         // Should only be overridden for tests
         notificationCenter: NotificationCenter = NotificationCenter.default,
         webLocale: Locale = Locale.autoupdatingCurrent,
-        authenticatedWebViewManager: AuthenticatedWebViewManager = .init()
+        authenticatedWebViewManager: AuthenticatedWebViewManager = .init(),
+        financialConnectionsPresenter: FinancialConnectionsPresenter = .init()
```

Comment:
I'm starting to second guess whether these should have default values or not. I'm worried that one day we will do a handoff to a new ConnectComponentWebViewController instance and not provide the existing managers.

Reply:
If we end up having that future case, then I agree we should stop configuring default values here since it could lead to issues. But given that we don't have that case today and these are only explicitly set for test injection, I'm not too concerned about it right now.